src/components/molecules/ModalBodyInfo.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+	<div class="body-container">
+		<div class="icon">
+			<slot name="icon"></slot>
+		</div>
+		<h4>{{ text }}</h4>
```

Comment:
Again, semanticly it's more a h1 or h2
```suggestion
		<h2 class="h4">{{ text }}</h2>
```